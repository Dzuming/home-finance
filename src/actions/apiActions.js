import * as types from './actionTypes';

export const apiRequestPending = () => ({
  type: types.API_REQUEST.PENDING,
});
export const apiRequestSuccess = () => ({
  type: types.API_REQUEST.SUCCESS,
});
