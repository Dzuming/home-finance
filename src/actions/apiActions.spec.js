import * as actions from './apiActions';
import expect from 'expect';

describe('api actions', () => {
  it('should create API_REQUEST_PENDING action with snapshot', () => {
    expect(actions.apiRequestPending()).toMatchSnapshot();
  });

  it('should create API_REQUEST_SUCCESS action with snapshot', () => {
    expect(actions.apiRequestSuccess()).toMatchSnapshot();
  });
});
