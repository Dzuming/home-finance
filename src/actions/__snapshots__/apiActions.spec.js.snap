// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`api actions should create API_REQUEST_PENDING action with snapshot 1`] = `
Object {
  "type": "API_REQUEST_PENDING",
}
`;

exports[`api actions should create API_REQUEST_SUCCESS action with snapshot 1`] = `
Object {
  "type": "API_REQUEST_SUCCESS",
}
`;
